openexr-rs/src/lib.rs

```diff
@@ -57,6 +57,8 @@ pub use flat_image_level::{
 };
 pub mod flat_image_channel;
 pub use flat_image_channel::{FlatChannelF16, FlatChannelF32, FlatChannelU32};
+pub mod version;
+pub use version::{TILED_FLAG, LONG_NAMES_FLAG, NON_IMAGE_FLAG, MULTI_PART_FILE_FLAG, ALL_FLAGS, Version};
```

Comment:
The other advantage of using bitflags is that it generates the flags as const members of a struct so they're scoped like `VersionFlags::TILED` rather than importing `TILED_FLAG` into the global namespace